pyproject.toml

```diff
@@ -76,8 +76,8 @@ cartopy = ["lib/cartopy/tests/mpl/baseline_images/**",
 write_to = "lib/cartopy/_version.py"
 
 [tool.codespell]
-ignore-words-list = "damon,koordinates,linz,slippy,subtiles,tring"
-skip = "./.git,./docs/build,./docs/source/gallery,./docs/source/reference,*.cpp,*.css,*.examples,*.js,*.html,*.ipynb,*.pdf,*.rst.txt"
+ignore-words-list = "damon,koordinates,linz,manuel,slippy,subtiles,tring"
+skip = "./.git,./docs/build,docs/source/copyright.rst,./docs/source/gallery,./docs/source/reference,*.cpp,*.css,*.examples,*.js,*.html,*.ipynb,*.pdf,*.rst.txt"
```

Comment:
Should this be `./docs/...` to match the others, or should the others drop the `./`?

Reply:
I tried locally with `./docs` and still got failures, which is why I switched. I do not know if the others are used or not...

Reply:
OK, I removed the other ones since they are unnecessary.